pythran/pythonic/types/list.hpp

```diff
@@ -580,6 +580,14 @@ namespace types
     erase(x);
     return res;
   }
+  template <class T>
+  void list<T>::clear()
+  {
+    long sz = size();
```

Comment:
why not just `_data->clear()` ?

Reply:
ah ah ah yes, I did not realize that _data was a std::vector! Of course.